e2e-pw/src/oss/poms/panels/embeddings-panel.ts

```diff
@@ -0,0 +1,75 @@
+import { Locator, Page, expect } from "src/oss/fixtures";
+import { EventUtils } from "src/shared/event-utils";
+import { SelectorPom } from "../selector";
+import { PanelPom } from "./panel";
+
+export class EmbeddingsPom {
+  readonly locator: Locator;
+  readonly plotContainer: Locator;
+  readonly selector: SelectorPom;
+  readonly colorbySelector: SelectorPom;
+  readonly asserter: EmebddingsAsserter;
+  readonly pp: Locator;
+  readonly lassoTool: Locator;
+
+  constructor(
+    private readonly page: Page,
+    private readonly eutils: EventUtils
+  ) {
+    this.pp = page;
+    this.locator = this.page.getByTestId("embeddings-container");
+    this.selector = new SelectorPom(this.locator, eutils, "embeddings");
+    this.colorbySelector = new SelectorPom(
+      this.locator,
+      eutils,
+      "embeddings-colorby"
+    );
+
+    this.plotContainer = this.page.getByTestId("embeddings-plot-container");
+    this.asserter = new EmebddingsAsserter(this, new PanelPom(page));
+    this.lassoTool = this.locator.getByTestId("embeddings-plot-option-lasso");
+  }
+}
+
+class EmebddingsAsserter {
+  constructor(
+    private readonly emPom: EmbeddingsPom,
```

Comment:
nit: Could we just call it `embeddingsPom`?

Reply:
haha sounds good mr. naming police ;)